src/docfx/docset/Document.cs

```diff
@@ -71,6 +81,11 @@ internal class Document
         /// </summary>
         public string RedirectionUrl { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether if it's an experimental content
```

Comment:
>whether if [](start = 36, length = 10)

`whether` == `if`

Reply:
thanks!

---
In reply to: [206508053](https://github.com/dotnet/docfx/pull/3127#discussion_r206508053) [](ancestors = 206508053)